tests/test_resolvers/test_stack_output.py

```diff
@@ -50,6 +50,16 @@ def test_resolver(self, mock_get_output_value):
             sceptre_role="dependency_sceptre_role",
         )
 
+    @patch("sceptre.resolvers.stack_output.StackOutput._get_output_value")
+    def test_resolver__badly_formatted(self, mock_get_output_value):
```

Comment:
I don't understand what this is testing. Where is the bad formatting?

Reply:
I have tried to make this a bit clearer.